evm_arithmetization/src/cpu/kernel/interpreter.rs

```diff
@@ -305,11 +305,23 @@ impl<'a, F: Field> Interpreter<'a, F> {
             (GlobalMetadata::BlockGasLimit, metadata.block_gaslimit),
             (GlobalMetadata::BlockChainId, metadata.block_chain_id),
             (GlobalMetadata::BlockBaseFee, metadata.block_base_fee),
+            (
+                GlobalMetadata::BlockBlobBaseFee,
+                metadata.block_blob_base_fee,
+            ),
```

Comment:
Unrelated, but another tiny oversight of https://github.com/0xPolygonZero/plonky2/pull/1313